CONTRIBUTING.md

```diff
@@ -1,6 +1,26 @@
+# Development "-dev" Versions and Dockerfile
+
+On the master branch and other non-release branches, the version number will end with `-dev`.
+On master, this should start from the latest release version number.
+
+These dev versions will not be published to PyPI or Dockerhub, and so must be build locally.
+
+The primary docker images for testing and development should be built locally as follows:
+```
+version=$(python -c "import armory; print(armory.__version__)")
+docker build --build-arg armory_version=${version} --target armory-tf1-dev -t twosixarmory/tf1:${version} .
+docker build --build-arg armory_version=${version} --target armory-tf2-dev -t twosixarmory/tf2:${version} .
+docker build --build-arg armory_version=${version} --target armory-pytorch-dev -t twosixarmory/pytorch:${version} .
+```
+NOTE: if the current version is not `-dev`, it may overwrite an existing release Dockerfile,
+only using the local repository instead of pulling from PyPI. This is desired behavior,
```

Comment:
So if we recommend `build-dev.sh` then that shell file won't be available if they pull from pypi (only whats under armory/ or specified in a manifest file)

Reply:
Then we should probably recommend that they download a release from github and build locally, because the Dockerfiles will also not be available.

Reply:
Note: this is the CONTRIBUTING.md file. Who would contribute but pull from pypi instead of github?

Reply:
I modified README.md to have users pull the code from the release on github.